tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -361,6 +361,13 @@
   "DD_TRACE_ACTIVITY_LISTENER_ENABLED": "trace_activity_listener_enabled",
   "DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED": "trace_128_bits_id_enabled",
   "DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED": "trace_128_bits_id_logging_enabled",
+  "DD_TRACE_HEALTH_METRICS_ENABLED": "dd_trace_health_metrics_enabled",
+  "DD_SPAN_SAMPLING_RULES_FILE": "dd_span_sampling_rules_file",
+  "DD_TRACE_COMPUTE_STATS": "dd_trace_compute_stats",
+  "DD_EXCEPTION_DEBUGGING_ENABLED": "dd_exception_debugging_enabled",
+  "DD_TRACE_OBFUSCATION_QUERY_STRING_PATTERN": "dd_trace_obfuscation_query_string_pattern",
+  "DD_CALL_BASIC_CONFIG": "dd_call_basic_config",
+  "DD_SERVICE_MAPPING": "dd_service_mapping",
```

Comment:
I assume these are updates from the dd-go version right? If so, thanks! 😃 

don't forget to make a PR to dd-go to add your new config values :)

Reply:
Yes it s been updated already :)